__main__.py

```diff
@@ -60,25 +60,16 @@ def conversation_name(conv):
     return ', '.join(u.full_name for u in users)
 
 
-def prompt_and_send_callback(loop, client, conversations):
-    def prompt_and_send():
+def send_message_callback(event, client, conversation):
```

Comment:
Two comments on naming:

- Though this callback is specifically called as a response to receiving a message, the naming suggests this callback is executed upon receiving a message. Perhaps `on_message_callback`? (I think the reason the previous name worked is because it wasn't called in response to anything...)

- When this callback is executed, you _know_ the event you're responding to is a `ChatMessageEvent`, so you should specify that in the name of the parameter: `event` -> `message_event`.